web/src/CityContentSwitcher.tsx

```diff
@@ -144,6 +145,13 @@ const CityContentSwitcher = ({ languageCode }: CityContentSwitcherProps): ReactE
 
       {tuNewsEnabled && render(TU_NEWS_ROUTE, TuNewsPage)}
       {tuNewsEnabled && render(TU_NEWS_DETAIL_ROUTE, TuNewsDetailPage)}
+
+      {(localNewsEnabled || tuNewsEnabled) && (
+        <Route
+          path={NEWS_ROUTE}
+          element={<Navigate to={localNewsEnabled ? LOCAL_NEWS_ROUTE : TU_NEWS_ROUTE} replace />}
+        />
+      )}
```

Comment:
Too align behavior between native and web. And because I initially thought that we will only support features, not specific slugs (i.e. `news` instead of `news/local`). Since it doesn't hurt and allows for a little shorter urls I would keep it. Do you disagree?

Reply:
Nah, sounds good, I was just wondering.